Plugins/Android/backgrounddownload/build.gradle

```diff
@@ -1,33 +1,13 @@
-buildscript {
-    repositories {
-        google()
-        jcenter()
-    }
-    dependencies {
-        classpath 'com.android.tools.build:gradle:3.2.0'
-    }
-}
-
-apply plugin: 'android-library'
+apply plugin: 'com.android.library'
 
 android {
-    sourceSets {
-        main {
-            manifest.srcFile 'AndroidManifest.xml'
-            java.srcDirs = ['src']
-            res.srcDirs = ['res']
-            assets.srcDirs = ['assets']
-            jniLibs.srcDirs = ['libs']
-        }
-    }
-
     compileSdkVersion 28
-    buildToolsVersion '28.0.3'
-    defaultConfig {
-        targetSdkVersion 16
-    }
 
-    lintOptions {
-        abortOnError false
+
+    defaultConfig {
+        minSdkVersion 16
+        targetSdkVersion 28
+        versionCode 1
+        versionName "1.0"
```

Comment:
Are the versions required for a library?

Reply:
Auto-added when creating library, will try to remove them, since we don't need those.

Reply:
Yup, we don't need those, removed.